CHANGELOG.md

```diff
@@ -12,6 +12,8 @@
   [Ilya Puchka](https://github.com/yonaskolb)
   [#178](https://github.com/stencilproject/Stencil/pull/178)
 
+- Added support for dynamic filter using `filter` filter
```

Comment:
If (when?) #221 gets merged, could you fix the format of this entry? Remove the newline above, add a `.` and 2 spaces at the end, and credit yourself + link to this PR.